mcs/class/System.Net.Http/System.Net.Http/HttpClient.wasm.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Reflection;
+
+namespace System.Net.Http {
+	public partial class HttpClient {
+
+        public static Func<HttpMessageHandler> GetHttpMessageHandler;
```

Comment:
This should be private. We cannot add public members to BCL types.

Reply:
Oh! Forgot one thing. We'll need to add GetHttpMessageHandler to the linker xml for System.Net.Http.
@akoeplinger can you help us with this?

Reply:
How does the variable get set if it is private?

Reply:
It would be set by reflection.

Reply:
:-)

Reply:
you can add the `[PreserveDependency ("GetHttpMessageHandler", "System.Net.Http.HttpClient")]` attribute to GetDefaultHandler().

Reply:
Thanks @akoeplinger